f21/ps.html

```diff
@@ -1269,39 +1272,30 @@ <h2><u> Lecture / Lab 9</u></h2>
         <p><b>REVIEW from Lecture/Lab 4 & 7</b>
-    <p><img src="../images/irmaScreenshot.png" width=400>
+    <p><img src="../images/python_calculator.png" width=400>
 
-    <p>Fill in the missing function to animate hurricane data (inspired by the 2018 <a href="http://nifty.stanford.edu/2018/ventura-hurricane-tracker/nifty-hurricanes.html" target="_blank">Nifty Hurricane Program</a> by Phil Ventura).
-      <p>Your function, <tt>animate(t,lat,lon,wind)</tt> takes as input:
+    <p>Fill in the missing function to a simple python calculator.
+      <p>Your function, <tt>main</tt> takes as input:
       <ul>
-        <li> <tt>t</tt>: a turtle,
-          <li> <tt>lat</tt>: an integer storing the current latitude,
-          <li> <tt>lon</tt>: an integer storing the current longitude, and
-          <li> <tt>wind</tt>: the current wind speed in miles per hour.
+        <li> <tt>select</tt>: a number for the operation,
+          <li> <tt>number_1</tt>: the first integer, and
+          <li> <tt>number_2</tt>: the second integer.
           </ul>
 
       <p>
-      Your function should move the turtle to the current location (longitude, latitude), and then based on the
-      <a href="https://www.nhc.noaa.gov/aboutsshws.php" target="_blank">Saffir-Simpson Hurricane Wind Scale</a>, change the turtle to be:
+      You should fill in the fucntions below that are blank in order for the calculator to run:
       <ul>
-        <li> red and pen size 5 for Category 5 (windspeed > 157 mph)
-        <li> orange and pen size 4 for Category 4  (windspeed in 130-156 mph)
-        <li> yellow and pen size 3 for Category 3   (windspeed in 111-129 mph)
-        <li> green and pen size 2 for Category 2   (windspeed in 96-110 mph)
-        <li> blue and pen size 1 for Category 1   (windspeed in 74-95 mph)
-        <li> white and pen size 1 if not hurricane strength
+	      <li> <tt>add(num1, num2)</tt> that takes two numbers and returns the result of their addition
+	      <li> <tt>subtract(num1, num2)</tt> that takes two numbers and returns the result of their subtraction
+	      <li> <tt>multiply(num1, num2)</tt> that takes two numbers and returns the result of their multiplication
+	      <li> <tt>divide(num1, num2)</tt> that takes two numbers and returns the result of their division
+	      <li> <tt>sqrt(num1)</tt> that takes a number and returns the square root of that number
+	      <li> <tt>squared(num1)</tt> that takes a number and returns the result of that number squared
+	      <li> <tt>exponent(num1, num2)</tt> that takes two numbers and returns the result of thr first number to the exponent of the second number
       </ul>
 
-      <p>A template program, <tt>hurricane.py</tt>, and background image, <tt>mapNASA.gif</tt>, are available on the
```

Comment:
Please include the paragraph "A template program, python_calculator.py, is available ..." and link to the github repo. Also please issue a pull request to the CSci 127 github repo to add python_calculator.py template there (the one with the functions to be filled out". Thank you!